raresGabriel01/home-assignment
# [ISSUE-31] t-grid virtualization

# Issue link
#31 
# Description
Implemented virutalization for the `t-grid` component using `CdkVirtualScrollViewport` from `@angular/cdk/scrolling`
# How to test?

1. Open the "large data set" story in the storybook
2. Scroll. Ensure the scrolling experience is smooth.
3. Inspect element and check if there is a small number of `<tr>` elements (only the ones visible within the DOM) in the DOM, and not 2000.

# Attachments
